fs/Kconfig

```diff
@@ -121,11 +121,17 @@ config FS_HEAPSIZE
 
 config FS_REFCOUNT
 	bool "File reference count"
-	default !DEFAULT_SMALL
```

Comment:
Not all developers are aware of DEFAULT_SMALL, this change will only keep the behavior before #13296 was introduced so that it does not cause any burden to other developers, the default configuration adds 500+ bytes per platform, which is unfair to junior developers.